src/org/labkey/targetedms/TargetedMSModule.java

```diff
@@ -207,7 +207,7 @@ public String getName()
     @Override
     public double getVersion()
     {
-        return 19.193;
+        return 19.21;
```

Comment:
There is another PR that's going to be merged before this one that also has an upgrade script. Please update this PR's version to 19.22.

Reply:
https://github.com/LabKey/targetedms/pull/29#

Reply:
done